composer.json

```diff
@@ -17,5 +17,8 @@
     ],
     "require-dev": {
         "phpunit/phpunit": "^7.5"
+    },
+    "require": {
+        "getdkan/contracts": "dev-loosen-storage-contracts"
```

Comment:
Once PR https://github.com/GetDKAN/contracts/pull/10#partial-pull-merging gets merge, the above should be switched back to master